pkg/operation/botanist/nginxingress.go

```diff
@@ -63,6 +63,8 @@ func (b *Botanist) DefaultNginxIngress() (component.DeployWaiter, error) {
 		component.ClusterTypeShoot,
 		externalTrafficPolicy,
 		v1beta1constants.ShootNginxIngressClass,
+		[]string{b.Shoot.GetIngressFQDN("*")},
```

Comment:
I didn't find the use of this in the case of `ClusterTypeShoot`.
```suggestion
		nil,
```

Reply:
Good point. This is indeed not required. Changed with https://github.com/gardener/gardener/pull/9038/commits/5a5d31af42a290c656cfd11770ce6b8bd460add2.